.github/workflows/dependency-review.yml

```diff
@@ -14,7 +17,14 @@ jobs:
   dependency-review:
     runs-on: ubuntu-latest
     steps:
-      - name: 'Checkout Repository'
-        uses: actions/checkout@v3
-      - name: 'Dependency Review'
+      - name: Checkout the repository
+        uses: actions/checkout@v3.0.2
+
+      - name: GitHub dependency vulnerability check
         uses: actions/dependency-review-action@v2
+
+      - name: FOSSA dependency license check
+        run: |
+          curl --no-progress-meter -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/fossa-cli/v${FOSSA_CLI_VERSION}/install-latest.sh | bash
+          fossa analyze --fossa-api-key ${{ secrets.MELTYBOT_FOSSA_API_KEY }} --revision ${{ github.sha }}
+          fossa test --fossa-api-key ${{ secrets.MELTYBOT_FOSSA_API_KEY }} --revision ${{ github.sha }}
```

Comment:
Looks like they have a github action: https://github.com/fossas/fossa-action

Apologies in advance if you already considered this.